File: src/main/java/superstitio/powers/masoOnly/LostBodyPart_Castration.java
package superstitio.powers.masoOnly;

import superstitio.DataManager;

import static superstitio.cards.general.FuckJob_Card.BodyPart;

//去势
public class LostBodyPart_Castration extends LostBodyPart {
    public static final String POWER_ID = DataManager.MakeTextID(LostBodyPart_Castration.class);

    public LostBodyPart_Castration() {
        super(POWER_ID);
    }

    @Override
    public BodyPart[] banedBodyPart() {
        return new BodyPart[]{BodyPart.genital};
    }
}

File: superstitioMain/java/superstitio/powers/masoOnly/LostBodyPart_Castration.kt
package superstitio.powers.masoOnly

import superstitio.DataManager
import superstitio.cards.general.FuckJob_Card.BodyPart

//去势
class LostBodyPart_Castration : LostBodyPart(POWER_ID)
{
    override fun banedBodyPart(): Array<BodyPart?>
    {
        return arrayOf(BodyPart.genital)
    }

    companion object
    {
        val POWER_ID: String = DataManager.MakeTextID(LostBodyPart_Castration::class.java)
    }
}
